src/main/java/no/nav/fo/veilarbregistrering/db/arbeidssoker/ArbeidssokerRepositoryImpl.java

```diff
@@ -65,4 +67,15 @@ public Arbeidssokerperioder finnFormidlingsgrupper(Foedselsnummer foedselsnummer
 
         return map(arbeidssokerperioder);
     }
+
+    @Override
+    public Arbeidssokerperioder finnFormidlingsgrupper(Bruker bruker) {
+        Arbeidssokerperioder arbeidssokerperioder = finnFormidlingsgrupper(bruker.getGjeldendeFoedselsnummer());
+
+        List<Arbeidssokerperioder> historiskeArbeidssokerperioder = bruker.getHistoriskeFoedselsnummer().stream()
+                .map(this::finnFormidlingsgrupper)
```

Comment:
I stedet for å loope igjennom alle FNR her, så kunne en vel ha gjort en _SELECT * FROM FORMIDLINGSGRUPPE WHERE FOEDSELSNUMMER = ?_, så kunne vi vel ha hørt en _SELECT * FROM FORMIDLINGSGRUPPE WHERE FOEDSELSNUMMER IN (?)_ (eller noe - husker ikke helt syntaksen) - så om det er 1 eller flere har ingen betydning for spørringen, og vi har 1 spørring for begge use-case. Da kunne vi hatt en metode på Bruker som gav en liste over identer.

Reply:
Etter noen forsøk får jeg ikke til å bruke et java-array som input til spørringen, så lar det være for nå.

Vil ikke utelukke at det er mulig å få til.